### PR TITLE
Rename octoapp to src

### PR DESCRIPTION
**Pull request by Ycandido0119**
### Asana Ticket

_NA_

### PR Description

Rename the octoapp directory to src, the reason for this is because there is another directory called octoapp. This is a refactor change with no change to code.

### Checklist

[ ] Went through and checked that I haven't commited anything I shouldn't.
[ ] I have gone over the commit with a mentor.
[ ] I have understood the logic of the ticket and if not wrote questions on things I am stuck on.

### CODE REVIEW

File: src/challenges/management/commands/my_custom_command.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            with open(file_path, "r") as file:
+                for line in file:
+                    split_line = line.strip("|\n").split("|")
+                    self.stdout.write(str(split_line))
+                    if split_line[0] == "026":
+                        dictionary = dict(zip(line_026, split_line[1:]))
+                        meter_point, created = MeterPoint.objects.get_or_create(
+                            mpan=dictionary["mpan"], BSC=dictionary["BSC"]
+                        )
+                        print(dictionary)
```
Comment (todaatsushi) [inline src/challenges/management/commands/my_custom_command.py:30]: Could remove these `print` calls while we're here 👀

File: src/challenges/management/commands/my_custom_command.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                    elif split_line[0] == "028":
+                        dictionary = dict(zip(line_028, split_line[1:]))
+                        meter, created = Meter.objects.get_or_create(
+                            serial_number=dictionary["serial_number"],
+                            meter_point=meter_point,
+                        )
+                        print(dictionary)
```
Comment (todaatsushi) [inline src/challenges/management/commands/my_custom_command.py:37]: ♻️

File: src/challenges/management/commands/my_custom_command.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                        else:
+                            meter_register = meter_register[1]
+                        Reading.objects.create(
+                            date=date,
+                            value=dictionary["value"],
+                            flow_filename=file_path,
+                            meter_register=meter_register,
+                        )
+
+                        print(dictionary)
```
Comment (todaatsushi) [inline src/challenges/management/commands/my_custom_command.py:56]: ♻️

**Review by todaatsushi — COMMENTED** [review]
As long as everything works, this looks good - do the tests pass / app functionality still work?

Only note is the README will need updating

File: README.md [hunk 4]
```diff
@@ -39,7 +39,7 @@ python -m venv .env
 Once you have created your virtual environment you will be able to see it by looking at the beginning of a line on your command prompt. Now we need to source into the virtual environment using this command
 
 ```bash
-source mysite/bin/activate
+source .venv/bin/activate
```
Comment (todaatsushi) [inline README.md:42]: This name doesn't match the env name above - this name is the more correct one as `.env` is a commonly used filename for holding sensitive vars.

**Review by todaatsushi — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (Ycandido0119) [reply README.md:42]: Okay am changing the on at the top to .venv

**Review by todaatsushi — APPROVED** [review]

**Pull request merged**